tests/Unit/PointwiseFunctions/AnalyticData/GrMhd/TestFunctions.py

```diff
@@ -2,10 +2,13 @@
 # See LICENSE.txt for details.
 
 import numpy as np
+from AnalyticSolutions.RelativisticEuler import TestFunctions as re
```

Comment:
I think better import the functions you need directly. We don’t have guidelines on this yet so this is only a suggestion, but I think it makes it easier to maintain this code.